src/subdomains/supporting/payin/entities/crypto-input.entity.ts

```diff
@@ -233,7 +225,7 @@ export class CryptoInput extends IEntity {
 
   completed() {
     this.status = PayInStatus.COMPLETED;
-    this.sendType = null;
+    this.action = null;
```

Comment:
Should we remove the action reset?

Reply:
Yes for what reason that was implemented? 😅

Reply:
I have no idea 😆

Reply:
Is somebody filtering on this?

Reply:
I will take a look on this 👍

Reply:
No seems not to be important. There is no filtering or anything on that I would say 👍